python/pyspark/sql/dataframe.py

```diff
@@ -4217,15 +4217,18 @@ def cov(self, col1: str, col2: str) -> float:
     def crosstab(self, col1: str, col2: str) -> "DataFrame":
         """
         Computes a pair-wise frequency table of the given columns. Also known as a contingency
-        table. The number of distinct values for each column should be less than 1e4. At most 1e6
-        non-zero pair frequencies will be returned.
```

Comment:
I think we can just remove this and don't add `.. versionchanged:: 3.4.0`. It's sort of improvement, and trivial to mention.

Reply:
got it